modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/ParsedParameter.java

```diff
@@ -19,7 +20,7 @@
  *
  * @since 3.7.0
  */
-final class ParsedParameter
+final class ParsedParameter implements AnnotatedElement
```

Comment:
Because ParameterModel implemented the method getAnnotation() (which is contained in the Interface AnnotatedElement) but not the other 2 methods I implemented in this commit. By implementing AnnotatedElement I could ask the ExpressionSupport of the ParsedParameter. See line https://github.com/mulesoft/mule/pull/2396/files#diff-3e704c3bd995eae525b12afc05490bddR413